gateware/cas.py

```diff
@@ -10,9 +10,10 @@
 from migen.genlib.misc import WaitTimer
 
 from litex.soc.cores.gpio import GPIOIn, GPIOOut
+from gateware.led import RGBLed
 
 class ControlAndStatus(Module, AutoCSR):
-    def __init__(self, platform, clk_freq):
+    def __init__(self, platform, clk_freq, rgb_pwm=False):
```

Comment:
What is with you and random boolean flags given to modules? :-P

I think we probably want to consider if we want pwm for *all* LEDs rather than only *just* for rgb LEDs?

Do you think you could split the RGB led and PWM parts of this pull request apart?